### ISSUE TITLE
PULL_REQUEST_FROM_BRANCH value not being passed

### ISSUE THREAD

**Issue by humblewebdev** [tomasbjerre/pull-request-notifier-for-bitbucket#151]
I'm trying to invoke a Jenkins GET using the format:

http://server/job/theJob/buildWithParameters?token=TOKEN&PARAMETER=${PULL_REQUEST_FROM_BRANCH}

I have this setup as a string parameter named PULL_REQUEST_FROM_BRANCH (default value=develop) in Jenkins.

My branch specifier is set to */${PULL_REQUEST_FROM_BRANCH}

when I create a merge request, Jenkins is building develop.

If I reset the default value to null, then the param Jenkins reports is null. 

Long story short, I don't think this variable is being passed by your BitBucket plugin.

Am I doing it wrong, or is this a legit bug?

**Comment by tomasbjerre** [comment]
Can you trigger Jenkins with this parameter with a curl command? There are examples in the readme.

**Comment by humblewebdev** [comment]
Oops. My bad. I forgot to replace boilerplate with an actual value. I feel stupid. thanks for the quickness of your response, and helping me get it right!

**Comment by tomasbjerre** [comment]
No problem :)
